index.html

```diff
@@ -87,10 +86,7 @@ <h3>Some Capstones</h3>
     <ul class="capstonelist">
     <li>
       <div class="topcapstone">2022 Bootcamp Web Development
-        <a href="" aria-label="link to IT education company neuefische">neuefische</a><br>
-        <figure id="neuefischelogo">
-          <img src="images/neue_fische_Logo.JPG" alt="logo IT education company neuefische, Hamburg">
-        </figure>
+        <a href="https://www.neuefische.de/" aria-label="link to IT education company neuefische" target="_blank"><img src="images/neue_fische_Logo.JPG" alt="logo IT education company neuefische, Hamburg"></a>
```

Comment:
if you want to create an cv(lebenslauf)...check this website for `table` [tabelle](https://css-tricks.com/complete-guide-table-element/) this helped me a lot ;) maybe a table is better in this case!?

Reply:
tbh i didn t want to spend a lot of time to create a table in html for this 5 points atm - if time will remain today maybe i try to build one in a separate branch, many thanks for that useful link 🙏 😊

> hey tanja! nice work so far ;) It´s responsive 👍 a few things...in the html i would recommend to wrap the skills section in the beginning in a `section` tag...like you already did with biography etc. then check your id´s again...normally the id is unique (eg for the sections contact, skills, biography) i would recommend class tags instead (eg change the `li` tag with id="notwrap" to class="notwrap") go one and good work so far ;)

Hi Michi, a lot of thanks for this great review with all the good advices. Awesome, pointing out the topic "id and classes" to me, i changed this as you suggested (hopefully i haven t forgotten one, but i will also look now better at this) and you are absolutely right.